kratos/utilities/time_discretization.cpp

```diff
@@ -55,6 +56,29 @@ void BDF::SetAuxBDFPointer(
     }
 }
 
+void BDF::ComputeBDFCoefficients(ModelPart &rModelPart) const
```

Comment:
```suggestion
void BDF::ComputeBDFCoefficients(ProcessInfo& rProcessInfo const
```

there is no need for passing the ModelPart

Maybe the fct name should then be ComputeAndSave...

Reply:
I thought it would be easier for the user to do it in this way, which is also equivalent to the old `ComputeBDFProcess`. Should this one be substituted by a `ComputeAndSaveBDFCoefficients(ProcessInfo &rProcessInfo)` or they can coexist?

Reply:
I would suggest not to have both methods, to keep it lightweight.

The effort for users to update their code is minimal in this case I would say

Reply:
OK. Let's go for the `ComputeAndSave...(ProcessInfo &rProcessInfo)`